alica_engine/include/engine/collections/Variant.h

```diff
@@ -1,5 +1,4 @@
 #pragma once
-#include "engine/blackboard/BBIdent.h"
 #include <assert.h>
```

Comment:
can kill this and use std::variant (in a different pr cc: @veerajsk

Reply:
sure. Wrike task: https://www.wrike.com/open.htm?id=799633896 . cc; @bjoernschroeder